.github/workflows/deploy.yml

```diff
@@ -52,6 +52,7 @@ jobs:
             --no-progress
             --base "$(pwd)/build"
             --accept 100..399,401,403,429
+            --exclude 'https://conda-forge/status/'
```

Comment:
To fix link check, is this what we want? Or should this be changed somehow?

Reply:
```suggestion
            --exclude '^.*/status/migration/.*$'
```

I think this should work. Let's see.